configure.ac

```
@@ -654,12 +654,14 @@ case $host in
          dnl It's safe to add these paths even if the functionality is disabled by
          dnl the user (--without-wallet or --without-gui for example).
 
-         openssl_prefix=`$BREW --prefix openssl 2>/dev/null`
-         bdb_prefix=`$BREW --prefix berkeley-db4 2>/dev/null`
-         qt5_prefix=`$BREW --prefix qt5 2>/dev/null`
-         if test x$openssl_prefix != x; then
-           PKG_CONFIG_PATH="$openssl_prefix/lib/pkgconfig:$PKG_CONFIG_PATH"
-           export PKG_CONFIG_PATH
+         if test x$enable_bip70 != xno; then
+           openssl_prefix=`$BREW --prefix openssl 2>/dev/null`
+           bdb_prefix=`$BREW --prefix berkeley-db4 2>/dev/null`
+           qt5_prefix=`$BREW --prefix qt5 2>/dev/null`
```

this doesn't look right imo, should be more like
```suggestion
         bdb_prefix=`$BREW --prefix berkeley-db4 2>/dev/null`
         qt5_prefix=`$BREW --prefix qt5 2>/dev/null`
         if test x$enable_bip70 != xno; then
           openssl_prefix=`$BREW --prefix openssl 2>/dev/null`
```

---

applied the commit, seems to work well